### DISCUSSION TITLE
Testing a multi-site Craft CMS project without a custom domain

### DISCUSSION THREAD

**Discussion by jamesmacwhite** [craftcms/cloud#39] [Q&A] [answered]
Is it possible to test a multi-site Craft CMS project hosted on Craft Cloud without adding a custom domain for initial testing?

**Comment by timkelty** [comment, marked as answer]
Yes, but not multi-domain, as you currently only get one preview domain.
So, you could set up multi-site with path-based base URLs:

- `https://my-site-0123abc.preview.craft.cloud/en`
- `https://my-site-0123abc.preview.craft.cloud/fr`

As long as you're properly linking to things using `url` functions in your templates, when you're ready for domains, you can simply switch over the base URLs to be domain-based.

There is an open request multiple preview domains [here](https://github.com/craftcms/cloud/discussions/23), which would allow you to test multi-site using preview domains.

**Reply by jamesmacwhite** [reply]
Thanks for this, it is what I thought due to the single preview domain, you can't have another temporary domain that is separate currently.

For multi-site specifically I would add my name to the request around allowing for multiple temporary/preview names, because of this scenario.
